packages/storage/src/bucket.js

```diff
@@ -697,6 +620,123 @@ Bucket.prototype.deleteLabels = function(labels, callback) {
 };
 
 /**
+ * <div class="notice">
+ *   <strong>Early Access Testers Only</strong>
+ *   <p>
+ *     This feature is not yet widely-available.
+ *   </p>
+ * </div>
+ *
+ * Disable `requesterPays` functionality from this bucket.
+ *
+ * @param {function=} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * bucket.disableRequesterPays(function(err) {
+ *   if (!err) {
+ *     // requesterPays functionality disabled successfully.
+ *   }
+ * });
+ *
+ * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * bucket.disableRequesterPays().then(function(data) {
+ *   var apiResponse = data[0];
+ * });
+ */
+Bucket.prototype.disableRequesterPays = function(callback) {
+  this.setMetadata({
+    billing: {
+      requesterPays: false
+    }
+  }, callback || common.util.noop);
+};
+
+/**
+ * <div class="notice">
+ *   <strong>Early Access Testers Only</strong>
+ *   <p>
+ *     This feature is not yet widely-available.
+ *   </p>
+ * </div>
+ *
+ * Enable `requesterPays` functionality for this bucket.
```

Comment:
I'd recommend providing more background for requester pays for clarity as it has two workflows. Workflow 1 is meant for the owner of the bucket and workflow 2 for requests made by non-owners.